sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -76,33 +76,18 @@ export function getTraceParentHeader(spanContext: SpanContext): string | undefin
 export function getTracer(): Tracer;
 
 // @public
-export type HrTime = [number, number];
+export function getTracer(name: string, version?: string): Tracer;
```

Comment:
Since we got rid of setTracer, what's the benefit of keeping getTracer around?

Reply:
The OTel method requires a name for the tracer to fetch, so it's incompatible as is with our current calls to `getTracer` in core-http and core-rest-pipeline.

I know we are looking to remove all of these shortly, but until we do (and take a dependency on opentelemetry/api directly) I figured I can just reuse `getTracer` and default to `azure/core-tracing` if it's not provided.